bundles/org.openhab.binding.nobohub/src/main/java/org/openhab/binding/nobohub/internal/NoboHubBindingConstants.java

```diff
@@ -0,0 +1,116 @@
+
+    public static final String PROPERTY_SOFTWARE_VERSION = "softwareVersion";
+
+    public static final String PROPERTY_ZONE = "zone";
+    public static final String PROPERTY_ZONE_ID = "id";
+    public static final String PROPERTY_TEMPERATURE_SENSOR_FOR_ZONE = "temperatureSensorForZone";
+
+    public static final int NOBO_HUB_TCP_PORT = 27779;
+
+    public static final Duration TIME_BETWEEN_FULL_SCANS = Duration.ofMinutes(10);
+    public static final Duration TIME_BETWEEN_RETRIES_ON_ERROR = Duration.ofSeconds(10);
+
+    public static final Duration RECOMMENDED_KEEPALIVE_INTERVAL = Duration.ofSeconds(14);
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_HUB = new ThingTypeUID(BINDING_ID, "nobohub");
+    public static final ThingTypeUID THING_TYPE_ZONE = new ThingTypeUID(BINDING_ID, "zone");
+    public static final ThingTypeUID THING_TYPE_COMPONENT = new ThingTypeUID(BINDING_ID, "component");
+
+    public static final Set<ThingTypeUID> AUTODISCOVERED_THING_TYPES_UIDS = new HashSet<>(
+            Arrays.asList(THING_TYPE_ZONE, THING_TYPE_COMPONENT));
+
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = new HashSet<>(
+            Arrays.asList(THING_TYPE_HUB, THING_TYPE_ZONE, THING_TYPE_COMPONENT));
+
+    // List of all Channel ids
+
+    // Hub
+    public static final String CHANNEL_HUB_ACTIVE_OVERRIDE_NAME = "activeOverrideName";
+
+    // Zone
+    public static final String CHANNEL_ZONE_ACTIVE_WEEK_PROFILE_NAME = "activeWeekProfileName";
+    public static final String CHANNEL_ZONE_ACTIVE_WEEK_PROFILE = "activeWeekProfile";
+    public static final String CHANNEL_ZONE_CALCULATED_WEEK_PROFILE_STATUS = "calculatedWeekProfileStatus";
+    public static final String CHANNEL_ZONE_COMFORT_TEMPERATURE = "comfortTemperature";
+    public static final String CHANNEL_ZONE_ECO_TEMPERATURE = "ecoTemperature";
+    public static final String CHANNEL_ZONE_CURRENT_TEMPERATURE = "currentTemperature";
+
+    // Component
+    public static final String CHANNEL_COMPONENT_CURRENT_TEMPERATURE = "currentTemperature";
+
+    // Date/time
+    public static final DateTimeFormatter DATE_FORMAT_SECONDS = DateTimeFormatter.ofPattern("yyyyMMddHHmmss");
+    public static final DateTimeFormatter DATE_FORMAT_MINUTES = DateTimeFormatter.ofPattern("yyyyMMddHHmm");
+    public static final DateTimeFormatter TIME_FORMAT_MINUTES = DateTimeFormatter.ofPattern("HHmm");
+
+    // Discovery
+    public static final int NOBO_HUB_BROADCAST_PORT = 10000;
+    public static final String NOBO_HUB_BROADCAST_ADDRESS = "0.0.0.0";
+    public static final int NOBO_HUB_MULTICAST_PORT = 10001;
+    public static final String NOBO_HUB_MULTICAST_ADDRESS = "239.0.1.187";
+
+    // Mappings
+
+    public static final Map<String, String> REJECT_REASONS = Stream.of(new String[][] {
+            { "0", "Client command set too old, run it in with debug logs and let the maintainer know" },
```

Comment:
"run it with debug logs" (without "in"), perhaps even "please run it with debug logs"? Same for next few texts.